purchase_order_report_with_note/__init__.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+#   Copyright (c) 2011 Camptocamp SA (http://www.camptocamp.com)
+#   @author Guewen Baconnier
```

Comment:
I'd remove this from init files. https://github.com/OCA/maintainer-tools/pull/260 But it's not blocking anyway